mako/compat.py

```diff
@@ -141,10 +141,7 @@ def exception_as():
     else:
         import dummy_thread as thread  # noqa
 
-if win32 or jython:
-    time_func = time.clock
-else:
-    time_func = time.time
+time_func = timeit.default_timer
```

Comment:
will this work in Python 2?   also at the moment we unfortunately haven't yet officially dropped Python 2.6 or Python versions < 3.3, see #249 .    these things will likely need a version 1.1 bump.   

alternatively, if you add some conditionals here, that would allow this to go right in.    I can run CI tests with that change, they will fail right now at least for python 2.7.

Reply:
> will this work in Python 2? also at the moment we unfortunately haven't yet officially dropped Python 2.6 or Python versions < 3.3, see #249 . these things will likely need a version 1.1 bump.

timeit.default_timer() exists since 2.3 (https://hg.python.org/cpython/rev/cd21be82f50b), but was only added to the docs with 2.7 (https://bugs.python.org/issue13478) so it should just work.

Reply:
OK lets try it